Tests/Entity/PostManagerTest.php

```diff
@@ -69,11 +68,52 @@ public function assertPostEnabled($qb, $flag)
         $qb->expects($this->once())->method('setParameters')->with($this->equalTo(array('enabled' => $flag)));
     }
 
+    public function testFindOneByPermalinkSlug()
+    {
+        $permalink = $this->getMock('Sonata\NewsBundle\Permalink\PermalinkInterface');
```

Comment:
this should be createMock (not sure if this was like this on 3.x, and should be fixed there first)